int16 HeadHorizontal
int16 HeadVertical
